conda/gateways/repodata/jlap/fetch.py

```diff
@@ -304,7 +304,10 @@ def request_url_jlap_state(
                     )
                 else:
                     raise JlapSkipZst()
-            except (JlapSkipZst, HTTPError) as e:
+            except (JlapSkipZst, HTTPError, zstandard.ZstdError) as e:
+                if isinstance(e, zstandard.ZstdError):
+                    # will this include the token
```

Comment:
I think that's reasonable, so maybe sanitize it? Or leave out the URL?

Reply:
Improved.